averbis/core/_rest_client.py

```diff
@@ -2045,6 +2187,193 @@ def get_project(self, name: str) -> Project:
         return Project(self, name)
 
     @experimental_api
+    def delete_resource_container(self, name: str) -> None:
+        """
+        HIGHLY EXPERIMENTAL API - may soon change or disappear.
+        Delete the named global resource container
+        """
+        client_base_url = "/experimental/textanalysis"
+        self._delete_resource_container(name, client_base_url)
+
+    @experimental_api
+    def _delete_resource_container(self, name: str, base_url: str) -> None:
+        """
+        HIGHLY EXPERIMENTAL API - may soon change or disappear.
+        Use the respective public delete_resource_container() method.
+        """
+        build_version = self.get_build_info()
+        if build_version["platformVersion"] < "8.23":
```

Comment:
We have a helper function `_is_higher_equal_version` that should be used instead of a string comparison.
The problem with a string comparison is that a version `8.3` would count as larger than `8.23`.
This also affects a fewer other places.